instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/http/HttpServerAttributesExtractor.java

```diff
@@ -10,7 +10,6 @@
 import io.opentelemetry.api.common.AttributesBuilder;
 import io.opentelemetry.context.Context;
 import io.opentelemetry.instrumentation.api.instrumenter.AttributesExtractor;
-import io.opentelemetry.instrumentation.api.instrumenter.net.NetServerAttributesGetter;
```

Comment:
I don't understand why you removed the import and just use the lengthy package name below. You did this in several places, so it looks intentional....it's just not clear to me how/why this helps the deprecation effort.

Reply:
Errorprone requires that when you use deprecated types.
I don't like this either, but I can kinda agree with that: using deprecated classes should look ugly, to encourage you to refactor the code that uses them